molecule/mariadb-version/INSTALL.rst

```diff
@@ -0,0 +1,16 @@
+*******
```

Comment:
I think this file can be removed.

Reply:
Yes, tests are working without this file.